config/development.js

```diff
@@ -0,0 +1,7 @@
+import R from 'ramda';
+
+import defaultConfig from './default';
+
+const config = {};
+
+export default R.mergeDeepRight(defaultConfig, config);
```

Comment:
This is neat. Could add a comment explaining what the expected `config` should look like after `mergeDeepRight` occurs, since it's not immediately obvious or a unit test?

Reply:
Right! Also `config` here is only used in testing, so `ramda` should be fine. but later if we want to adopt it into api, may need to replace ramda with a smaller package in considering of sdk bundling size~